Review-08-Numerical-Integration-and-Differentiation.ipynb

```diff
@@ -270,6 +270,15 @@
     "\n",
     "The n-point Gaussian quadrature rule corresponds to the zeros of an orthogonal polynomial."
    ]
+  },
+   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 8.24. (a) What does it mean for a sequence of quadrature rules to be progressive?\n(b) Why is this property important?\n",
+    "\n",
+    "(a) It means that the nodes of rules can be reused for subsequent rules.\n(b) This is important because it reduces the need to evaluate the function as new points for error estimation."
```

Comment:
I am going to submit a small change to reword (b).

from
> This is important because it reduces the need to evaluate the function as new points for error estimation.

to
> This is important because it minimizes the computation required to evaluate an integrand by reusing rules from previous computation when evaluating the function at new points.